guides/release/components/the-component-lifecycle.md

```diff
@@ -54,9 +54,9 @@ you can use `didUpdateAttrs` to clear any error state that was built up from edi
   {{/each}}
 </ul>
 <fieldset>
-  {{input name="user.name" value=this.name change=(action "required")}}
-  {{input name="user.department" value=this.department change=(action "required")}}
-  {{input name="user.email" value=this.email change=(action "required")}}
+  <Input @name="user.name"       @value={{this.name}}       @change={{action "required"}} />
```

Comment:
`@change={{this.required}}`?
w/
```ts
@action required() {}
```
?

Reply:
We are not changing to native classes and decorators yet.

Reply:
@locks - Should probably fix the whitespace here, I don't think we should be padding like this (padding for alignment is 👿)...